1.0/docs/devguide/templates.md

```diff
@@ -303,6 +303,28 @@ If you or a third-party library mutate the array without Polymer's methods,
 you need to call [`notifySplices`](#notifysplices) to ensure that any elements
 watching the array are properly notified.
 
+### Improve performance for large lists {#large-list-perf}
```

Comment:
Extra credit: the #notifySplices link 2 lines up is broken, and should probably be another API docs link... :D